activestorage/test/test_helper.rb

```diff
@@ -4,11 +4,15 @@
 require "active_support"
 require "active_support/test_case"
 require "active_support/testing/autorun"
-require "byebug"
+
+begin
+  require "byebug"
+rescue LoadError
```

Comment:
Byebug doesn't get installed in CI. This matches what Action Cable does:

https://github.com/rails/rails/blob/8f44d1661981a57838543b8a9890c855d7b02b89/actioncable/test/test_helper.rb#L10-L13